pkg/commands/compute/build.go

```diff
@@ -192,7 +192,8 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 			contl := strings.ToLower(cont)
 			if contl != "y" && contl != "yes" {
 				text.Info(out, "Stopping the build process.")
-				return nil
+				text.Break(out)
+				return fsterr.ErrBuildStopped
```

Comment:
Is there any way to test this case?

Reply:
I actually had to update the tests to make them pass by ensuring I set an expectation for an error to be returned (see my last commit)